src/extension/tracker/index.ts

```diff
@@ -225,8 +223,15 @@ async function loginToTracker(): Promise<void> {
       },
     );
 
+    nodecg().log.debug('[Tracker] Login response:', {
+      status: resp2.statusCode,
+      body: resp2.body,
+      headers: resp2.headers,
+      cookies: resp2.cookies,
+    });
+
     // If we're not being redirected or there's no session token, the login failed.
-    if (resp2.statusCode !== 302 || (resp2.cookies && !resp2.cookies.tracker_session)) {
+    if (resp2.statusCode !== 302 || (resp2.cookies && !resp2.cookies.sessionid)) {
```

Comment:
To be quite honest, abusing the cookies from another domain (that of the tracker) in the environment of the streamgraphics is kinda sus. I understand it's because of the inherited codebase, but I feel like I should mention it. (even first getting the csrf token feels shady, doesn't the tracker expose some sort of API?)

Reply:
Yes it does have an api for that, the api is authenticated via the session.

Reply:
No, that's not how this works. The thing first gets the actual html for the login page, which contains the csrf token necessary for submitting form contents. An API would allow for submitting requests without such a need, but we're getting into semantics here (and this could even be the way django works, but I wouldn't know as I don't have experience with it)